plugins/generator-1.15.2/forge-1.15.2/procedures/entity_spawn_x.java.ftl

```diff
@@ -0,0 +1,14 @@
+(new Object() {
+	public int getSpawnX(IWorld _world, Entity _ent) {
+		int retval = _world.getSpawnPoint().getX();
+		if (_ent instanceof PlayerEntity) {
+			BlockPos _bp = ((PlayerEntity) _ent).getBedLocation(_world.getDimension().getType());
+			if (_bp != null)
+				return _bp.getX();
```

Comment:
You could just assign retval here (`retval = _bp.getX();`) and then omit all else statements and return retval at the end, to shorten and clean up the code. Applies to all templates in pr.

Reply:
I don't understand what you mean. Could you write down the entire code here? (only for X position, I'll copy for Y and Z)

Reply:
I will edit this file from github to indicate what I meant, if indentations are wrong, please correct them as github handles them a bit funny.

Reply:
It is no performance change, but a little bit cleaner

Reply:
Edited

Reply:
Okay, I'll check them out.

> It is no performance change, but a little bit cleaner

Sure, I get that.